### PR TITLE
Feat: create task to update indexing of worlds

### PR DESCRIPTION
**Pull request by braianj**
To do this we should follow [this](https://github.com/decentraland/dcl-name-stats#should-index-endpoint) steps

- Create migration with hidden property and updated indexs
- Create function to return indexed and not indexed worlds
- Update check scene task to apply new property hidden to places and worlds
- Create task to check everyday indexed worlds

### CODE REVIEW

File: src/entities/World/task/verifyWorldsIndexing.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+export async function verifyWorldsIndexing(names: string[]) {
+  const shouldIndexFetch = await fetch(
+    "https://dcl-name-stats.decentraland.org/should-index",
+    {
+      body: JSON.stringify({ dclNames: names }),
+      method: "POST",
+    }
+  )
```
Comment (lauti7) [inline src/entities/World/task/verifyWorldsIndexing.ts:14]: Shouldn't be a `try {} catch(e) {}`?

**Review by lauti7 — APPROVED** [review]
LGTM, just one comment

File: src/entities/Place/model.ts [hunk 2]
```diff
@@ -342,4 +343,46 @@ export default class PlaceModel extends Model<PlaceAttributes> {
...
+
+    return this.namedQuery("find_worlds", sql)
+  }
+
+  static updateWorldsShown = (names: string[]) => {
+    if (names.length === 0) {
+      return 0
+    }
+
+    const sqlNames = SQL`${join(names.map((item) => SQL`${item}`))}`
```
Comment (2fd) [inline src/entities/Place/model.ts:361]:
```suggestion
```

File: src/entities/Place/model.ts [hunk 3]
```diff
@@ -342,4 +343,46 @@ export default class PlaceModel extends Model<PlaceAttributes> {
...
+    `
+    return this.namedRowCount("update_world_shown", sql)
+  }
+
+  static updateWorldsHidden = (names: string[]) => {
+    if (names.length === 0) {
+      return 0
+    }
+
+    const sqlNames = SQL`${join(names.map((item) => SQL`${item}`))}`
```
Comment (2fd) [inline src/entities/Place/model.ts:377]:
```suggestion
```

File: src/entities/World/task/verifyWorldsIndexing.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import fetch from "node-fetch"
+
+type ShouldIndexResponseProps = {
+  data: { dclName: string; shouldBeIndexed: boolean }[]
+}
+
+export async function verifyWorldsIndexing(names: string[]) {
+  try {
+    const shouldIndexFetch = await fetch(
+      "https://dcl-name-stats.decentraland.org/should-index",
```
Comment (2fd) [inline src/entities/World/task/verifyWorldsIndexing.ts:10]:
```suggestion
      env("WORLDS_INDEX_API", "https://dcl-name-stats.decentraland.org/should-index")
```

File: src/entities/World/task/verifyWorldsIndexing.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      {
+        body: JSON.stringify({ dclNames: names }),
+        method: "POST",
+      }
+    )
+    const shouldIndex: ShouldIndexResponseProps = await shouldIndexFetch.json()
+    const shouldIndexNames = shouldIndex.data
+      .map((world) => world.shouldBeIndexed === true && world.dclName)
+      .filter((world) => !!world) as string[]
```
Comment (2fd) [inline src/entities/World/task/verifyWorldsIndexing.ts:19]:
```ts
    const shouldIndexNames = shouldIndex.data
      .filter((world) => world.shouldBeIndexed === true)
      .map((world) => world.dclName)
```

👀

File: src/entities/World/task/verifyWorldsIndexing.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+    const shouldNotIndexNames = shouldIndex.data
+      .map((world) => world.shouldBeIndexed !== true && world.dclName)
+      .filter((world) => !!world) as string[]
```
Comment (2fd) [inline src/entities/World/task/verifyWorldsIndexing.ts:23]:
```ts
    const shouldNotIndexNames = shouldIndex.data
      .filter((world) => world.shouldBeIndexed !== true)
      .map((world) => world.dclName)
```

:eyes:

File: src/entities/World/task/processWorldsIndexing.ts [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  )
+
+  if (worldsVerified.hasIndexNames) {
+    const updateWorlds = await PlaceModel.updateWorldsShown(
+      worldsVerified.indexNames
+    )
+    loggerExtended.log(`Worlds updated to show ${updateWorlds}`)
+  }
+
+  if (worldsVerified.hasNonIndexNames) {
```
Comment (2fd) [inline src/entities/World/task/processWorldsIndexing.ts:29]: couldn't this be done with only one query?

**Review by 2fd — CHANGES_REQUESTED** [review]

File: src/entities/Place/model.ts [hunk 8]
```diff
@@ -3,6 +3,7 @@ import {
   SQL,
   columns,
   conditional,
+  join,
```
Comment (2fd) [inline src/entities/Place/model.ts:6]:
```suggestion
```

**Review by 2fd — APPROVED** [review]

**Pull request merged**